src/permitInfo/fetchPermitInfo.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * Fetch permit info for given network
+ *
+ * ChainId is the required first cli argument
+ *
+ * TokenListPath is the optional second cli argument
+ * By default, checks against the respective default token list
+ * - `src/public/CowSwap.json` for mainnet and gnosis chain
+ * - `src/public/CowSwapGoerli.json` for goerli
+ *
+ * RpcUrl is the optional third cli argument
+ * By default, it'll use Infura for mainnet and goerli.
+ * In that case, `INFURA_API_KEY` env var must be set.
+ *
+ * Minimal example:
+ * $ ts-node fetchPermitInfo 100
+ *
+ * Minimal example using default INFURA rpc:
+ * $ INFURA_API_KEY=0000000...111 ts-node fetchPermitInfo 1
+ *
+ * With optional params
+ * $ ts-node fetchPermitInfo 1 otherTokenList.json https://my.rpc.endpoint
+ *
+ * @arg chainId - required, first positional argument
+ * @arg tokenListPath - optional, second positional argument
+ * @arg rpcUrl - optional, third positional argument
+ */
+
+import {getTokenPermitInfo, PermitInfo} from '@cowprotocol/permit-utils'
+import * as path from 'node:path'
+import {readFileSync, writeFileSync} from 'node:fs'
+import {JsonRpcProvider} from '@ethersproject/providers'
+import {argv, chdir, exit} from 'node:process'
+import {BASE_PATH, SPENDER_ADDRESS} from './const.js'
```

Comment:
Why those imports have .js ext? I see they are actually ts files

Reply:
Good question, I had not noticed this.
It was auto import by the IDE. Will fix it

Reply:
Hmmm 
<img width="1016" alt="image" src="https://github.com/cowprotocol/token-lists/assets/43217/e25ff4bb-dca6-4c15-bce0-cc8c48b698b9">

Reply:
Fixed!